django_orghierarchy/models.py

```diff
@@ -47,6 +47,7 @@ class Organization(models.Model):
     dissolution_date = models.DateField(blank=True, null=True, help_text=_('A date of dissolution'))
     parent = models.ForeignKey('self', null=True, blank=True,
                                help_text=_('The organizations that contain this organization'))
+    affiliated = models.ManyToManyField('self', blank=True, help_text=_('Affiliated organizations'))
```

Comment:
Hmm. Did we specify a need for one affiliated organization to have several parents?

I'm not sure if ManyToMany or just simple ForeignKey by the name `responsible_organization` is better for future use cases, or if it is needless complexity that an affiliated organization may have several superorganizations responsible for it?

Either way, we'll need to specify a descriptive name for the reverse relation, e.g. `responsible_organization` or something similar.

Reply:
For some reason, I got the impression (probably it's just my own thoughts :sweat_smile: ) from the kick-off meeting that an organization can be affiliated to one or more organizations. but yeah, that's one thing needs to be cleared.

Reply:
The field name is mainly borrowed from the comment: https://github.com/City-of-Helsinki/linkedevents/issues/234#issuecomment-334113087

Reply:
Yeah the field name is good, I'm thinking the reverse relation, and I think that might actually be the more important thing. So, instead of affiliated, we could have `responsible_organization` that is ForeignKey (like a parent, but not a real parent :D), and its reverse can be `affiliated_organizations`.